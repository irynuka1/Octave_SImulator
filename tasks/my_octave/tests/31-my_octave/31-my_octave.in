L 2 2
1 2
3 4
L 3 2
4 5
6 7
8 9
L 3 3
1 2 3
4 5 6
7 8 9
R 4 142214
R 0 15
R 2 456
O
D 1
T 0
R 2 -5
C 1
3
0 1 2
2
1 2
M 1 0
T 4
P 3
R 3 98425141
O
D 2
F 0
Q