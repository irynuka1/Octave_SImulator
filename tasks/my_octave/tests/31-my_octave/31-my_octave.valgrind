==12862== Memcheck, a memory error detector
==12862== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==12862== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==12862== Command: /home/irynuka1/Desktop/Programs/Tema2_1/check_my_octave_v1.1/check_my_octave/my_octave
==12862== Parent PID: 12861
==12862== 
==12862== 
==12862== HEAP SUMMARY:
==12862==     in use at exit: 0 bytes in 0 blocks
==12862==   total heap usage: 460 allocs, 460 frees, 14,828 bytes allocated
==12862== 
==12862== All heap blocks were freed -- no leaks are possible
==12862== 
==12862== For lists of detected and suppressed errors, rerun with: -s
==12862== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
