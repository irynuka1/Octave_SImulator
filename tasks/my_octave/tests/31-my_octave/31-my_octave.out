No matrix with the given index
3 3
Power should be positive
No matrix with the given index
7461 2006 6558 
7782 2050 6325 
8103 2094 6092 
3 3
